### DISCUSSION TITLE
[Build] [All] element plus 源码 dev 分支打包报错

### DISCUSSION THREAD

**Discussion by lixiaofa** [element-plus/element-plus#12866] [Q&A]
Bug Type: **`Build`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.3.4`
- Browser / OS: `105.0.5195.102（正式版本） （64 位）`
- Build Tool: `Vite`

### Related Component

- `All`

### Reproduction Link

[Github Repo](https://github.com/element-plus/element-plus.git)

### Steps to reproduce

pnpm build

### What is Expected?

正常打包， 不报错

### What is actually happening?

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

 ELIFECYCLE  Command failed with exit code 134.
[10:06:18] 'shellTask:generateTypesDefinitions' errored after 3.22 min

### Additional comments

_(empty)_

**Comment by lixiaofa** [comment]
![image](https://github.com/element-plus/element-plus/assets/23230215/ee9a06f0-2768-43ab-8d7c-8887704b0dce)
我的步骤： 
pnpm install
pnpm run build --max_old_space_size=8192
是我遗漏了步骤吗
各位大佬
@chenxch

**Comment by shixindea** [comment]
![image](https://github.com/user-attachments/assets/1267f40b-dd8d-454d-9bd1-0cacfdc44d74)
同问
